cumulusci/core/config/scratch_org_config.py

```diff
@@ -81,13 +81,26 @@ def create_org(self) -> None:
 
         def raise_error() -> NoReturn:
             message = f"{FAILED_TO_CREATE_SCRATCH_ORG}: \n{stdout}\n{stderr}"
+            try:
+                Output = json.loads(stdout)
```

Comment:
Variable names in Python typically use "`lower_snake_case`".

```suggestion
                output = json.loads(stdout)
```